src/cephadm/cephadm

```diff
@@ -2591,6 +2591,24 @@ def command_bootstrap():
                         get_fqdn(), port,
                         args.initial_dashboard_user,
                         password))
+    
+    if args.apply_spec:
+        logger.info('Applying %s to cluster' % args.apply_spec)
+
+        with open(args.apply_spec) as f:
+            for line in f:
+                if 'hostname:' in line:
+                    line = line.replace('\n', '')
+                    split = line.split(': ')
+                    if split[1] != host:
+                        logger.info('Adding ssh key to %s' % split[1])
+                        out, err, code = call_throws(['ssh-copy-id', '-f', '-i', '/etc/ceph/ceph.pub', 'root@%s' % split[1]])
+
+        mounts = {}
+        mounts[pathify(args.apply_spec)] = '/tmp/spec.yml:z'
+
+        out = cli(['orch', 'apply', '-i', '/tmp/spec.yml'], extra_mounts=mounts)
```

Comment:
instead of extra mounts, what about stdin?

Reply:
I give that a try this is how files were passed to the cli in other places though so i just went with that